j2k/src/org/jetbrains/kotlin/j2k/ExpressionConverter.kt

```diff
@@ -650,7 +650,7 @@ class DefaultExpressionConverter : JavaElementVisitor(), ExpressionConverter {
             result = MethodCallExpression.buildNonNull(operandConverted, typeConversion)
         }
         else {
-            val nullability = if (operandConverted.isNullable && !expression.isQualifier())
+            val nullability = if ((operandConverted.isNullable && !expression.isQualifier()) || operandType == PsiType.NULL)
```

Comment:
Better not to hack this check, but fix problem with `operandConverted.isNullable == false` for `null` literal